app/scripts/controllers/exportImport.js

```diff
@@ -126,6 +126,12 @@ angular.module('openhimConsoleApp')
       var exportData = angular.copy( $scope.selectedExports );
       var textFile = null;
 
+      // OHM-382: warn when exporting key
+      Alerting.AlertReset();
+      if(exportData !== undefined && exportData.Keystore !== undefined && exportData.Keystore.length > 0) {
```

Comment:
o, i think i should have made it !== null, even this has no impact! let re-run my HIM instance to check if i fall into the issue again that made me include this check